buffer/buffer_manager_test.go

```diff
@@ -37,7 +37,13 @@ var _ = Describe("Buffer Manager", func() {
 			// in the Fluent Bit code that we can't unit test
 			UseApiKey: true,
 		}
-		nrClient = nrclient.NewNRClient(nrClientConfig)
+		noProxy := config.ProxyConfig{}
+
+		var err error
+		nrClient, _ = nrclient.NewNRClient(nrClientConfig, noProxy)
```

Comment:
`err` not being set here?

Reply:
My bad, forgot to set it (I had not set it initially but then I finally decided to check it). Thanks for spotting this.